### DISCUSSION TITLE
Genius pro

### DISCUSSION THREAD

**Discussion by gabfus** [dgtaheno/artillery-Genius#9] [Q&A] [answered]
Hello your latest firmware 2.1.x with new input shaping work with my "genius pro" with auto bed leveling?
Wath is the max speed reached for pla or petg using input snaping?

Thanks

**Comment by dgtaheno** [comment, marked as answer]
Hello @gabfus ,

glad that it is working for your genius pro, not sure if it is 100% compatible...as it is prepared for artillery genius (not pro).
Auto bed levelling has not been configured in this software as it was not planned for the BL touch version.
As far as I know it depends on several factors(material, calibration, in summary how fine tuned is your printer).
In the other hand I´m not an expert on speed printing, but I recommend you to test it on your own increasing progressively the speed.

Best regards.

**Reply by dgtaheno** [reply]
Hi @gabfus ,

just created a Genius PRO version of the firmware, so I recommend you to try it.
https://github.com/dgtaheno/artillery-Genius-Genius-Pro-X1-Marlinfw/blob/Marlin-2.1.2.1/Marlin%202.1.2.1_PRO.bin
Your feedback would be appreciated here.

Best regards.
